### PR TITLE
Privacy fixes

### PR DESCRIPTION
**Pull request by inverse**
@alexesprit  can you test this on Firefox? I noticed my previous implementation only worked on chrome and thus is useless.

### CODE REVIEW

File: src/startup/startup.js [hunk 1]
```diff
@@ -29,19 +29,12 @@ require([
 		window.close();
 	}
 
-	chrome.runtime.getPackageDirectoryEntry((root) => {
```
Comment (inverse) [inline src/startup/startup.js]: This didn't work :cry:

File: src/core/background/main.js [hunk 2]
```diff
@@ -437,13 +437,14 @@ require([
...
 		});
-
-
 	}
-
 	startup();
 
-	chrome.runtime.onInstalled.addListener(function() {
+	chrome.runtime.onInstalled.addListener((object) => {
+		if ('install' !== object.reason) {
```
Comment (inverse) [inline src/core/background/main.js]: Prevent updates firing the new tab

**Comment by inverse** [comment]
@alexesprit tested locally seems fine - had to add some work around to give feedback since FF doesn't like window.close() when launched from `chrome.tabs.create` know a workaround?

**Comment by inverse** [comment]
@marsjaninzmarsa Interesting - how is the tabId determined?

**Comment by marsjaninzmarsa** [comment]
@inverse I was going to bet it's returned by `tab.create`, but checked docs and no - Promise is returned. Promise resolving to `Tab` object containing `id` property... c'mon, people, MDN don't hurt! 😂

**Pull request merged**